<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Region</class>
 <widget class="QDialog" name="Region">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="WelcomeLabelText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Region Setting for My Mosque</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="CountryListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAfterCurrent</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="CountryListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Country :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CityListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="CityListLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LongitudLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LatitudeLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LongitudTextbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LatitudeTextbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangingLanLonPush">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Advance Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPush">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>-10</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Languages/build-MyMosqu-Desktop-Debug/Untitled-1.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeZone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Zone :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="TimeZoneEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-25</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="CountryListLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the region of the mosque</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Language_List">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arabic</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="CountryListLabel_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Language :</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date :</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackPush">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
